packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -141,6 +141,54 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(LionListbox)) {
     ];
   }
 
+  /**
+   * @override ChoiceGroupMixin
+   */
+  get modelValue() {
```

Comment:
Could we do smth like:
```js
get modelValue() {
  if (!this.requireOptionMatch) {
    if (!this.multipleChoice) {
      return this._inputNode.value;
    } else {
      // This must be bound to a chip system or smth that would be maintained seperately...
      // It might be smth we want to implement later and for now we could throw an error on "!this.requireOptionMatch && this.multipleChoice"
    }
  }
  return super.modelValue;
}
```

Reply:
You want the `requireOptionMatch` to be the default, so non breaking?

`super.modelValue` does not exists.

Reply:
Seems this is inherited indirectly from ChoiceGroupMixin (via ListboxMixin)?